x-pack/plugins/integration_assistant/common/api/model/common_attributes.schema.yaml

```diff
@@ -36,6 +36,27 @@ components:
       items:
         type: object
 
+    SampleFormat:
```

Comment:
```suggestion
    SampleLogFormat:
```

For better understanding.

Reply:
But we had `rawSamples`, not `rawLogSamples`

Reply:
Sure , yeah. But I guess the idea of using `SampleFormat` is to identify the `format of samples` and not `a sample format`. So just to make this clear , I had the suggestion. But feel free to choose what you like to use.

Reply:
Perhaps `samplesFormat`? I just want consistency with the existing attributes.

Reply:
Sounds good. Yeah.